--- CMakeLists.txt	2018-10-31 08:07:22.000000000 -0700
+++ CMakeLists.txt	2019-12-04 21:50:07.606700200 -0800
@@ -1337,6 +1337,10 @@
   # Libraries for Win32 native and MinGW
   list(APPEND EXTRA_LIBS user32 gdi32 winmm imm32 ole32 oleaut32 version uuid advapi32 setupapi shell32)

+  if(MSVC AND NOT ${MSVC_VERSION} LESS 1920)
+    list(APPEND EXTRA_LIBS vcruntime)
+  endif()
+
   # TODO: in configure.ac the check for timers is set on
   # cygwin | mingw32* - does this include mingw32CE?
   if(SDL_TIMERS)
